CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.7.0 (Unreleased)
```

Comment:
Just pull all of the 1.6.2 changes up into this one. 1.6.2 was *going* to be the next version but now that we're doing a minor version bump it will never exist and all of the listed changes will be in the 1.7.0 version.

Reply:
ah, i see. will update.